flink-cdc-connect/flink-cdc-source-connectors/flink-connector-sqlserver-cdc/src/main/java/org/apache/flink/cdc/connectors/sqlserver/source/dialect/SqlServerChunkSplitter.java

```diff
@@ -61,11 +62,30 @@ public class SqlServerChunkSplitter implements JdbcSourceChunkSplitter {
     private final JdbcSourceConfig sourceConfig;
     private final JdbcDataSourceDialect dialect;
 
+    private static final String UNIQUEIDENTIFIRER = "uniqueidentifier";
+
     public SqlServerChunkSplitter(JdbcSourceConfig sourceConfig, JdbcDataSourceDialect dialect) {
         this.sourceConfig = sourceConfig;
         this.dialect = dialect;
     }
 
+    private static String reverse(String text) {
+        String[] arrs = text.split("-");
+        List<String> arrList = Arrays.asList(arrs);
+        Collections.reverse(arrList);
+        arrs = arrList.toArray(new String[0]);
+        return String.join("-", arrs);
+    }
+
+    public static int compare(Object obj1, Object obj2, Column splitColumn) {
+        if (splitColumn.typeName().equals(UNIQUEIDENTIFIRER)) {
+            String sObj1 = reverse(obj1.toString());
+            String sOjb2 = reverse(obj2.toString());
+            return ObjectUtils.compare(sObj1, sOjb2);
+        }
+        return ObjectUtils.compare(obj1, obj2);
+    }
```

Comment:
Will it be better if we rename them to `reverseUuid` / `compareUuid`, and move these utility functions to `org.apache.flink.cdc.connectors.sqlserver.source.utils` package?